File: java/CreditCardValidator.java
import java.util.Scanner;


public class CreditCardValidator{
public static void main(String[] args){

Scanner input = new Scanner(System.in);

System.out.print("Enter the length of your digits: ");
int num1 = input.nextInt();
if(num1 < 13 || num1 > 16){
System.out.print("Invalid length of number or invalid input");
}else{

int[] atm = new int [num1];

System.out.print("Enter each digit on your card: ");
for(int counter = 0; counter < num1; counter++){
atm[counter] = input.nextInt();
}
System.out.println();
System.out.println();

CreditCardValidatorFunction.validate(num1, atm);
}

}

}

File: python/CreditCardValidator.py
import CreditCardValidatorFunction;


print("Enter the length of your digits: ");
num1 = int(input());
if(num1 < 13 or num1 > 16):
	print("Invalid length of number or invalid input");
else:

	atm = [];

	print("Enter each digit on your card: ");
	for numbers in range(0, num1):
		num = int(input(f"Enter the digits on your card accordingly {numbers + 1}: "));
		if(num < 0 or num > 9):
			print("Invalid length of number or invalid input");
			break;
		else:

			atm.append(num);

	print();
	print();

validate(atm);




